<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLineEdit" name="Num_Mat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Taper le N° de matricule</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>65</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>N° Matricule       : </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Typedeconge">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox {
	border:2px solid #1921ab;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	font-family:Arial;
	font-size:11px;
	border-size:4px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
}</string>
    </property>
    <property name="title">
     <string>Type de Congé : </string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>19</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="CSS">
        <property name="text">
         <string>CSS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CE">
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CP">
        <property name="text">
         <string>CP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CR">
        <property name="text">
         <string>CR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>06h -&gt; 14h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14h -&gt; 22h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22h -&gt; 06h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aucun</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Temps du travail : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Valider">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>510</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#4579c7;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:11px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
	text-shadow:2px 1px 0px #2f6627;
}
.QPushButton:hover {
	background-color:#402abd;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>132</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Date début : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Icons/Icons/calandrier1-removebg-preview.png);
border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>125</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>AA-MM-JJ</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Jours">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>185</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>0.0</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Pour demande un demi jour taper ( 0.5 )</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>plusieur jours juste taper le nombre (ex : 4)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Durée demandé :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Temps du travail (prochaine): </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>06h -&gt; 14h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14h -&gt; 22h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22h -&gt; 06h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aucun</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
